CHANGELOG.md

```diff
@@ -4,13 +4,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+- [#65](https://github.com/alta/protopatch/pull/65): add optional support for proto3
```

Comment:
- Lines should end should end in a `.` if they are a sentence.
- Backtick quote `optional`.
- Should ideally link to the relevant documentation or release notes at the protobuf project on why `optional` support matters.